mmv1/third_party/terraform/services/integrationconnectors/resource_integration_connectors_endpoint_attachment_test.go

```diff
@@ -0,0 +1,77 @@
+package integrationconnectors_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccIntegrationConnectorsEndpointAttachment_integrationConnectorsEndpointAttachmentExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckIntegrationConnectorsEndpointAttachmentDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIntegrationConnectorsEndpointAttachment_integrationConnectorsEndpointAttachmentExample_full(context),
+			},
+			{
+				ResourceName:            "google_integration_connectors_endpoint_attachment.sampleendpointattachment",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "name", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccIntegrationConnectorsEndpointAttachment_integrationConnectorsEndpointAttachmentExample_update(context),
+			},
+			{
+				ResourceName:            "google_integration_connectors_endpoint_attachment.sampleendpointattachment",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "name", "labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccIntegrationConnectorsEndpointAttachment_integrationConnectorsEndpointAttachmentExample_full(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_project" "test_project" {
+}
+
+resource "google_integration_connectors_endpoint_attachment" "sampleendpointattachment" {
+  name     = "tf-test-test-endpoint-attachment%{random_suffix}"
+  location = "us-central1"
+  description = "tf created description"
+  service_attachment = "projects/connectors-example/regions/us-central1/serviceAttachments/test"
```

Comment:
Is this service attachment resource `projects/connectors-example/regions/us-central1/serviceAttachments/test` real, or is it a dummy value for the acceptance test to use? I'm asking because I'm not sure if it's a third party dependency or not.

Reply:
This is an example of what service attachment should be, to finish the LRO succesfully, this is a real resource.